src/Auth.js

```diff
@@ -4,7 +4,7 @@ import TwitterLogin from "react-twitter-login";
 const Auth = ({ addUser }) => {
   const authHandler = (error, data) => {
     if (error) {
-      alert("Please log into Twitter!");
+      alert(error);
```

Comment:
Alerting to the error to the user would not be helpful here. If everything is correctly configured on our end, the only reason that this should fail is the user did not authenticate.

Reply:
Yes, you are totally right. I changed it for testing purposes but will revert it back to the next PR.

Thank you for noticing!